bdk-ffi/src/descriptor.rs

```diff
@@ -257,14 +257,14 @@ mod test {
     use bdk::descriptor::DescriptorError::Key;
     use bdk::keys::KeyError::InvalidNetwork;
 
-    fn get_descriptor_secret_key() -> DescriptorSecretKey {
+    fn get_inner() -> DescriptorSecretKey {
```

Comment:
I think I told you to go ahead and change this one too, but now that I'm looking at it closer I think this name should stay `get_descriptor_secret_key()` since it's just a test helper function and is returning the bdk_ffi version of `DescriptorSecretKey`.